src/protocol/sort/reshare.rs

```diff
@@ -128,4 +136,43 @@ mod tests {
         }
         assert!(new_reshares_atleast_once);
     }
+
+    #[tokio::test]
+    pub async fn test_reshare_malicious() {
```

Comment:
```suggestion
    pub async fn reshare_malicious() {
```

Reply:
Its a test for reshare_malicious so needs to be named differently.